examples/rules_ios/.bazelrc

```diff
@@ -0,0 +1,29 @@
+# Import parent workspace settings
+import %workspace%/../../shared.bazelrc
+
+# https://github.com/bazel-ios/rules_ios/issues/672
+build --noexperimental_strict_conflict_checks
+
+# Exercise the extra flags feature
```

Comment:
I would make this the bare minimum (so see what is in Sanitizers?) and then add what is needed for rules_ios.